src/Teams/Data/ApplicationDbContext.cs

```diff
@@ -12,5 +10,8 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
             : base(options)
         {
         }
+
+        public DbSet<Team> team { get; set; }
```

Comment:
1. Create team table : team(id, team_name, team_owner)
2. Create team members table : team_members (id, team_id - FK, member_id)

Reply:
Also here just an example how it looks like, naming in database can be like this: TeamMembers, or team_members
Both fine, butt need to choose only one, not checked it yet, but you can check which names used right now for users in database.
For C# another naming convention

Reply:
Ok, thanks for the update. Even though the Style guidelines by MS allow both for camel- and snake-case, I should have stood to a more consistent style - i.e. camel-case since it has already been used in the project. Fixed that.